### PR TITLE
Add partial stage for pinning to Admiral

### PR DESCRIPTION
**Pull request by mkolesnik**
This stage will rlease Admiral and update consuming projects to use it.

Related to #31 

Signed-off-by: Mike Kolesnik <mkolesni@redhat.com>

### CODE REVIEW

File: scripts/release.sh [hunk 1]
```diff
@@ -72,6 +82,26 @@ function release_shipyard() {
...
+}
+
+function release_admiral() {
+    local project=admiral
+
+    # Release Admiral first so that we get the tag
+    clone_repo
+    create_project_release || errors=$((errors+1))
+
+    # Create a PR to pin Admiral on every one of it's consumers
```
Comment (dfarrell07) [inline scripts/release.sh:99]:
```suggestion
    # Create a PR to pin Admiral on every one of its consumers
```

**Review by dfarrell07 — APPROVED** [review]

File: scripts/lib/utils [hunk 2]
```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
 readonly PROJECTS=(admiral lighthouse shipyard submariner submariner-charts submariner-operator)
+readonly ADMIRAL_CONSUMERS=(lighthouse submariner)
```
Comment (mangelajo) [inline scripts/lib/utils:4]: submariner-operator also consumes admiral  now

Comment (mangelajo) [inline scripts/lib/utils:4]:
```suggestion
readonly ADMIRAL_CONSUMERS=(lighthouse submariner submariner-operator)
```

**Review by mangelajo — CHANGES_REQUESTED** [review]

File: scripts/lib/utils (hunk 2 above)
Comment (mkolesnik) [reply scripts/lib/utils:4]: > submariner-operator also consumes admiral now

Where do you see it? It's not in the go.mod

https://github.com/submariner-io/submariner-operator/blob/master/go.mod

Comment (mangelajo) [reply scripts/lib/utils:4]: ohh wait, I was confusing submariner with submariner-operator.... too much ovn work lately, sorry :D

**Review by mangelajo — APPROVED** [review]

**Comment by mangelajo** [comment]
Oppps I'm an idiot this was on top of the pre-releases patch

**Pull request merged**